homeassistant/components/airvisual/config_flow.py

```diff
@@ -36,6 +36,8 @@ class AirVisualFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
 
     def __init__(self):
         """Initialize the config flow."""
+        self._check_key_error_schema = None
+        self._check_key_error_step = None
         self._geo_id = None
```

Comment:
Why is this stored on `self`? We should only store it on `self` if we need it across two steps.